CriticalMass/UIView+Autolayout.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIView+Autolayout.swift
+//  CriticalMaps
+//
+//  Created by Leonard Thomas on 30.09.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    func addLayoutsCenter(in view: UIView, size: CGSize) {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        view.addConstraints([.init(item: self, attribute: .centerX, relatedBy: .equal, toItem: view, attribute: .centerX, multiplier: 1, constant: 1),
+                             .init(item: self, attribute: .centerY, relatedBy: .equal, toItem: view, attribute: .centerY, multiplier: 1, constant: 0)])
+        addConstraints([
+            .init(item: self, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: size.height),
+            .init(item: self, attribute: .width, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: size.width),
```

Comment:
I would use the anchor properties of the views to build the comstraints because to me they remove noise like the multiplier which is not used here